static/src/stylesheets/inline/story-package-garnett.scss

```diff
@@ -28,7 +34,37 @@
 }
 
 .fc-container--story-package {
-    background-color: #6b6e71; // This colour does not exist (in the palette)
+    background-color: $dynamo-background;
+
+    .fc-container__body {
+        &:before {
+            @include mq($from: leftCol) {
+                position: absolute;
+                content: '';
+                display: block;
+                top: 0;
+                bottom: -12px;
+                width: 1px;
+                background-color: #929698;
+                left: 170px; //magic numbers sorry --\_()_/--
```

Comment:
I think you might want to `position: relative` on `fc-container__body` and then `-10px` on the left value to pull this from the edge (and it'll then work on both breakpoints?)

Reply:
Done